src/Client.php

```diff
@@ -182,6 +182,13 @@ public function send(RequestInterface $request)
                 return new FutureResponse(new RejectedPromise($e));
             }
             throw RequestException::wrapException($trans->request, $e);
+        } catch (\TypeError $error) {
```

Comment:
I don't recall. Presumably I just ran our test suite and added a catch block for our specific failure. I don't see an immediate reason to not catch Throwable, so I'll update the PR.